src/language-server.ts

```diff
@@ -191,6 +191,21 @@ namespace LanguageServer {
 			Uri.joinPath(releaseUri, "releases", "0.1.0", "iex").fsPath,
 			0o755
 		);
+
+		const remoteControlDirectoryName = fs
+			.readdirSync(Uri.joinPath(releaseUri, "lib").fsPath)
+			.find((lib) => lib.match(/^remote_control/));
+
+		if (remoteControlDirectoryName !== undefined) {
+			const portWrapperUri = Uri.joinPath(
+				releaseUri,
+				"lib",
+				remoteControlDirectoryName,
+				"priv",
+				"port_wrapper.sh"
+			);
+			fs.chmodSync(portWrapperUri.fsPath, 0o755);
```

Comment:
interesting, after downloading, the the prot_wrapper.sh is not a executable sh.

Reply:
Yeah none of the files that should be executable are. Don't know why that is. It's definitely not a zip thing, maybe github makes them non-executable for security reasons or something 🤷